tests/integration/records/test_views.py

```diff
@@ -977,3 +1001,124 @@ def test_h_index_with_as_many_papers_as_citations(
+                                "key": "10.0-50.0",
+                                "from": 10.0,
+                                "to": 50.0,
+                                "doc_count": 0,
+                            },
+                            {
+                                "key": "50.0-100.0",
+                                "from": 50.0,
+                                "to": 100.0,
+                                "doc_count": 0,
+                            },
+                            {
+                                "key": "100.0-250.0",
+                                "from": 100.0,
+                                "to": 250.0,
+                                "doc_count": 0,
+                            },
+                            {
+                                "key": "250.0-500.0",
+                                "from": 250.0,
+                                "to": 500.0,
+                                "doc_count": 0,
+                            },
+                            {"key": "500.0-*", "from": 500.0, "doc_count": 0},
+                        ]
+                    },
+                    "average_citations": {"value": 8.0},
+                },
+                "published": {
+                    "doc_count": 0,
+                    "citations_count": {"value": 0.0},
+                    "citation_buckets": {
+                        "buckets": [
+                            {"key": "0.0-1.0", "from": 0.0, "to": 1.0, "doc_count": 0},
+                            {
+                                "key": "1.0-10.0",
+                                "from": 1.0,
+                                "to": 10.0,
+                                "doc_count": 0,
+                            },
+                            {
+                                "key": "10.0-50.0",
+                                "from": 10.0,
+                                "to": 50.0,
+                                "doc_count": 0,
+                            },
+                            {
+                                "key": "50.0-100.0",
+                                "from": 50.0,
+                                "to": 100.0,
+                                "doc_count": 0,
+                            },
+                            {
+                                "key": "100.0-250.0",
+                                "from": 100.0,
+                                "to": 250.0,
+                                "doc_count": 0,
+                            },
+                            {
+                                "key": "250.0-500.0",
+                                "from": 250.0,
+                                "to": 500.0,
+                                "doc_count": 0,
+                            },
+                            {"key": "500.0-*", "from": 500.0, "doc_count": 0},
+                        ]
+                    },
+                    "average_citations": {"value": None},
```

Comment:
is this correct?

Reply:
This is the behavior of the elastic search average aggregation. Basically, we are doing an average over 0 documents so they do not return 0 but None. We handle this on the UI though, where the average appears as 0. 
Here is a discussion on the elastic search thing https://github.com/elastic/elasticsearch/issues/31920